include/hip/hcc_detail/ocml_math_fwd.h

```diff
@@ -0,0 +1,431 @@
+float __ocml_fdim_f32(float, float);
+__attribute__((const))
+float __ocml_floor_f32(float);
+__attribute__((const))
+float __ocml_fma_f32(float, float, float);
+__attribute__((const))
+float __ocml_fmax_f32(float, float);
+__attribute__((const))
+float __ocml_fmin_f32(float, float);
+__attribute__((const))
+float __ocml_fmod_f32(float, float);
+float __ocml_frexp_f32(float, __attribute__((address_space(5))) int*);
+__attribute__((const))
+float __ocml_hypot_f32(float, float);
+__attribute__((const))
+int __ocml_ilogb_f32(float);
+__attribute__((const))
+int __ocml_isfinite_f32(float);
+__attribute__((const))
+int __ocml_isinf_f32(float);
+__attribute__((const))
+int __ocml_isnan_f32(float);
+float __ocml_j0_f32(float);
+float __ocml_j1_f32(float);
+__attribute__((const))
+float __ocml_ldexp_f32(float, int);
+float __ocml_lgamma_f32(float);
+__attribute__((pure))
+float __ocml_log10_f32(float);
+__attribute__((pure))
+float __ocml_log1p_f32(float);
+__attribute__((pure))
+float __ocml_log2_f32(float);
+__attribute__((const))
+float __ocml_logb_f32(float);
+__attribute__((pure))
+float __ocml_log_f32(float);
+float __ocml_modf_f32(float, __attribute__((address_space(5))) float*);
+__attribute__((const))
+float __ocml_nearbyint_f32(float);
+__attribute__((const))
+float __ocml_nextafter_f32(float, float);
+__attribute__((const))
+float __ocml_len3_f32(float, float, float);
+__attribute__((const))
+float __ocml_len4_f32(float, float, float, float);
+__attribute__((pure))
+float __ocml_ncdf_f32(float);
+__attribute__((pure))
+float __ocml_ncdfinv_f32(float);
+__attribute__((pure))
+float __ocml_pow_f32(float, float);
+__attribute__((pure))
+float __ocml_rcbrt_f32(float);
+__attribute__((const))
+float __ocml_remainder_f32(float, float);
+float __ocml_remquo_f32(float, float, __attribute__((address_space(5))) int*);
+__attribute__((const))
+float __ocml_rhypot_f32(float, float);
+__attribute__((const))
+float __ocml_rint_f32(float);
+__attribute__((const))
+float __ocml_rlen3_f32(float, float, float);
+__attribute__((const))
+float __ocml_rlen4_f32(float, float, float, float);
+__attribute__((const))
+float __ocml_round_f32(float);
+__attribute__((pure))
+float __ocml_rsqrt_f32(float);
+__attribute__((const))
+float __ocml_scalb_f32(float, float);
+__attribute__((const))
+float __ocml_scalbn_f32(float, int);
+__attribute__((const))
+int __ocml_signbit_f32(float);
+float __ocml_sincos_f32(float, __attribute__((address_space(5))) float*);
+float __ocml_sincospi_f32(float, __attribute__((address_space(5))) float*);
+float __ocml_sin_f32(float);
+__attribute__((pure))
+float __ocml_sinh_f32(float);
+float __ocml_sinpi_f32(float);
+__attribute__((const))
+float __ocml_sqrt_f32(float);
+float __ocml_tan_f32(float);
+__attribute__((pure))
+float __ocml_tanh_f32(float);
+float __ocml_tgamma_f32(float);
+__attribute__((const))
+float __ocml_trunc_f32(float);
+float __ocml_y0_f32(float);
+float __ocml_y1_f32(float);
+
+// BEGIN INTRINSICS
```

Comment:
These should all be removed.  Specially rounded operations are in OCML (but not working)

Reply:
Done, thanks.